src/test/java/core/basesyntax/LambdaCalcTest.java

```diff
@@ -0,0 +1,363 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class LambdaCalcTest {
+    private static final double DELTA = 0.0001;
+    private static final double FIRST_NUMBER = 153.1486;
+    private static final double SECOND_NUMBER = 153.1486;
+    private static final char ADDITION = '+';
+    private static final char SUBTRACTION = '-';
+    private static final char MULTIPLICATION = '*';
+    private static final char DIVISION = '/';
+    private static final char POWER = '^';
+    private static LambdaCalc calculator;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new LambdaCalc();
+    }
+
+    @Test
+    void additionWithTwoOPositiveOperands_Ok() {
```

Comment:
```suggestion
    void calculate_additionWithTwoOPositiveOperands_Ok() {
```
please add the name of the method that you are testing, to all test methods